include/WebServer.hpp

```diff
@@ -1,7 +1,7 @@
 #ifndef WEBSERVER_HPP
 #define WEBSERVER_HPP
 
-#define CONNECTION_TIMEOUT	18000
```

Comment:
Increased this again so we wont run out of time before even getting the blog up first